### ISSUE TITLE
Django 2.2 possible showstopper

### ISSUE THREAD

**Issue by ShaheedHaque** [django-polymorphic/django-polymorphic#382]
I believe that there is a showstopper incompatibility between Django 2.2 and django-polymorphic 2.0.3 as follows:

```
...
/usr/local/lib/python3.7/dist-packages/django/contrib/admin/options.py:1596: in _changeform_view
    media = media + inline_formset.media
/usr/local/lib/python3.7/dist-packages/django/contrib/admin/helpers.py:324: in media
    media = self.opts.media + self.formset.media
/usr/local/lib/python3.7/dist-packages/polymorphic/formsets/models.py:236: in media
    add_media(media, form.media)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

dest = Media(css={}, js=[]), media = Media(css={}, js=[])

    def add_media(dest, media):
        """
        Optimized version of django.forms.Media.__add__() that doesn't create new objects.
    
        Only required for Django < 2.0
        """
        if django.VERSION >= (2, 0):
            combined = dest + media
>           dest._css = combined._css
E           AttributeError: can't set attribute

combined   = Media(css={}, js=[])
dest       = Media(css={}, js=[])
media      = Media(css={}, js=[])

/usr/local/lib/python3.7/dist-packages/polymorphic/formsets/utils.py:15: AttributeError
```
This is because _css and _js are properties (I presume they were not previously):
```
    @property
    def _css(self):
        css = defaultdict(list)
        for css_list in self._css_lists:
            for medium, sublist in css_list.items():
                css[medium].append(sublist)
        return {medium: self.merge(*lists) for medium, lists in css.items()}

    @property
    def _js(self):
        return self.merge(*self._js_lists)
``` 
So - without knowing much about what this code is doing - I guess the code needs to change to use the _css_lists and _js_lists.

I could really use a fix for this, but would be happy to help with a PR if that would move things along. Please advise...

**Comment by ShaheedHaque** [comment]
FWIW, I assume the change in Django is related to [this](https://docs.djangoproject.com/en/2.2/releases/2.2/#merging-of-form-media-assets).

**Comment by ShaheedHaque** [comment]
See #379

**Comment by vdboor** [comment]
Fixed in 622e1e780b84a8e04c5af2d6758fb457ff92ea93!
